src/cljs/rems/administration/create_workflow.cljs

```diff
@@ -58,7 +58,7 @@
        (not (str/blank? (:title request)))
        (case (:type request)
          :auto-approve true
-         :dynamic (not (empty? (:handlers request)))
+         :dynamic (seq (:handlers request))
```

Comment:
this should return a boolean. IMO `(seq ...)` is a valid replacement for `(not (empty? ...))` only when the return value is used immediately

Reply:
I've reordered the statements so that it is not returned.